lib/msal-node/docs/brokering.md

```diff
@@ -55,6 +55,48 @@ pca.acquireTokenInteractive({
 });
 ```
 
+## Proof of Possession
+
+Access token proof of possession is supported when acquiring tokens through the native broker. To request a PoP token you will need to add a couple additional properties to the request object provided to `acquireTokenInteractive` or `acquireTokenSilent`
+
+### AT PoP Request Parameters
+
+|           Name          |                      Description                            | Required |
+|-------------------------| ----------------------------------------------------------- | -------- |
+|  `authenticationScheme` | Indicates whether MSAL should acquire a `Bearer` or `PoP` token. Default is `Bearer`. | **Required** |
+| `resourceRequestMethod` | The all-caps name of the HTTP method of the request that will use the signed token (`GET`, `POST`, `PUT`, etc.) | **Required** |
```

Comment:
I'd generally suggest we uppercase the incoming value rather than asking for all-caps input.